.golangci.yml

```diff
@@ -0,0 +1,61 @@
+# options for analysis running
+run:
+  # default concurrency is a available CPU number
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  timeout: 5m
+linters:
+  # please, do not use `enable-all`: it's deprecated and will be removed soon.
+  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
+  disable-all: true
+  enable:
+    - typecheck
+    - govet
+    - gosec
+#    - nolintlint
```

Comment:
Perhaps remove the commented out linters?

Reply:
Done